src/app/common/utils/sorts/common.util.sort,sortAlphabeticallyByProp.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { transactionCategoriesObjectsMockFunc } from 'src/app/domains/categories/domains.transaction-categories.mocks';
+import { TransactionCategory } from 'src/app/pages/categories/transaction-category.model';
+import { sortAlphabeticallyByProp } from './common.util.sort,sortAlphabeticallyByProp';
```

Comment:
Please fix file name `common.utils.sorts.ts` should be enough (there were also typos)